### PR TITLE
Tortollan Creature Trait

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Added a **_Tortollan_** creature trait.
  - Added an icon and a description for the trait.
  - Set lifespan to +10000. _("[Tortollans can live for thousands of years, with a 2,372-year-old apparently still being considered young.](https://wowpedia.fandom.com/wiki/Tortollan#Background)")_
  - Added almost every **_Tortollan_** character that was in WoW except some children from various locations, most notably [Little Tortollan](https://wowpedia.fandom.com/wiki/Little_Tortolla), [Tortaka Refuge](https://wowpedia.fandom.com/wiki/Tortaka_Refuge), and [Seekers' Expedition](https://wowpedia.fandom.com/wiki/Seekers%27_Expedition).
  - Changed the religion and culture of `c_darkwood`, `c_torga`, `c_dashazok and` `c_atalgral` to be **_Torgi_** and **_Tortollan_**, respectively.
- 3cd5d02e089c3b626068ea1d61a2d6c97a087d1c (**Apr 16 2023**):
  - Added 5 different **_Tortollan_** static models, each with 4 color variations.
  - Added three **_Tortollan_** dynasties: `dynn_Tortaka`, `dynn_von_Tortollan` and `Crabmaster`.
  - Replaced **Kiro** (ID: `381000`) with **Elder Kuppaka** (ID: `elder_kuppaka`) as the holder of `c_darkwood` starting `538.4.21`. This is the location of the [Tortaka tribe](https://wowpedia.fandom.com/wiki/Tortaka_tribe).
  - Replaced previous holder of `c_torga`, **Drukinjo** (ID: `66110`), with **Kisha** (ID: `kisha_tortollan`) and made her independent from `e_zandalari`. Tortollan are a neutral faction so I think this makes sense.
  - Made Loh the Ancient ruler of Zeb'ahari under `e_zandalari`. Made some Zandalari characters to populate his court.
  - Made **Ludwig von Tortollan** (ID: `ludwig_von_tortollan`) a courtier in **Freehold**.
- ff7bbb6a61145d4668e965bd21d5620df112b2d4 (**Apr 18 2023**):
  - Added **_Tortollan Child_** static model with 4 color variations.
- d11299f11594107254b30997814357b2967b22a5 (**Apr 18 2023**):
  - Added [Spiritsinger Umbra](https://wowpedia.fandom.com/wiki/Spiritsinger_Umbra?so=search) (ID: `umbra_tortollan`) as a character and made her the holder of `c_dashazok` in **Un'Goro Crater**.
  - Added `make_important_lore_character_effect` to notable **_Tortollan_** characters.
  - ~~Reverted `c_torga` being ruled by **Kisha** (ID: `kisha_tortollan`) since it doesn't make sense given the timeline. Need to rethink this.~~
  - Added the historical **_Tortollan_** nicknames _the Ancient_ and _the Hardened_ and assigned them to the appropriate characters.
  - Added custom traits to **Jol the Ancient**, **Loh**, **Ludwig von Tortollan**, and **Spiritsinger Umbra**.
  - Made **Jol the Ancient** a courtier of **Rastakhan**.
  - Made **Loh** chill in **Tel'Abim**.
- 0ec6f8e335c23b55a037fa82e157b1b29d1d1dcc (**Apr 19 2023**):
  - Assigned scripted static models to certain characters so that they match their appearance in WoW.
  - Added the `dynn_Hardshell` dynasty and its three members: **Gurtuk** (ID: gurtuk_hardshell), **Soola** (ID: `soola_hardshell`), and **Tooka** (ID: `tooka_hardshell`).
  - Created semi-canon relations between certain characters to make dynasties like **Hardshell** and **Tortaka** feel more connected (e.g. made some characters children or siblings of other characters).
  - Added traits to the following characters:
    -  **Kisha** (ID: `kisha_tortollan`)
    - **Lashk** (ID: `lashk_tortollan`)
    - **Maedin** (ID: `maedin_tortollan`)
    - **Majo** (ID: `majo_tortollan`)
    - **Roko** (ID: `roko_tortollan`)
    - **Toki** (ID: `toki_tortollan`)
    - **Torka** (ID: `torka_tortollan`)
    - **Churka** (ID: `churka_tortollan`)
    - **Elder Kuppaka** (ID: `elder_kuppaka`)
    - **Kurtu** (ID: `kurtu_tortollan`)
    - **Teekcha** (ID: `teekcha_tortollan`)
  - Made **Teekcha** and **Tulu** friends.
  - Made **Lashk** and **Kisha** friends.
  - Added **Roko** (ID: `roko_tortollan`) to **Daelin Proudmoore**'s court.
- 31fdd7a3462318f9a1ce2b3d262f88c1756d7b61 (**Apr 22 2023**):
  - Added some **_Tortollan_** dynasties to historical characters. Used a mix of [Tortle names](https://www.fantasynamegenerators.com/dnd-tortle-names.php) and compound noun names similar to [Hardshell](https://wowpedia.fandom.com/wiki/Gurtuk_Hardshell).
  - Assigned traits to **Rauloo the Hardened**, **Hansa**, and **Wahla**.
  - Added **Joma** (ID: `joma_tortollan`), Majo's son.

<details><summary>Click to view images (updated Apr 18, 2023)</summary>

image
image
image
image
image

</details>

## To-do list:
- [x] Localize all of the new names.
- [x] Add static models. (3cd5d02e089c3b626068ea1d61a2d6c97a087d1c)
- [x] Add static models for Tortollan children. (ff7bbb6a61145d4668e965bd21d5620df112b2d4)
- [x] Add the following nicknames:
  - [x] _the Hardened_
  - [x] _the Ancient_
  - [x] _the Scalemonger_
  - [x] _the Unkillable_
- [ ] _(Optional)_ Add the following titles with flavorization:
  - [x] Scrollsage (**Note**: Perhaps a title used by Tortollans with a high Learning stat.) -added as court position. Right now has the same effects as a court scholar, but available only to those with Chelonian heritage,
- [x] Add the following historical dynasties:
  - [x] _von Tortollan_ (3cd5d02e089c3b626068ea1d61a2d6c97a087d1c)
  - [x] _Tortaka tribe_ (3cd5d02e089c3b626068ea1d61a2d6c97a087d1c)
  - [x] Hardshell (0ec6f8e335c23b55a037fa82e157b1b29d1d1dcc)
- [x] Add Tortollan holders to the following counties:
  - [x] `c_darkwood` (3cd5d02e089c3b626068ea1d61a2d6c97a087d1c)
  - [x] _(Optional)_ `c_torga`
    - ~~Reverted in d11299f11594107254b30997814357b2967b22a5 since it doesn't make sense given the timeline. Need to rethink this.~~
- [x] Add employers to Tortollan character history files. Some of them didn't get employers to prevent overpopulating courts.
- [x] Adjust the traits of important Tortollan characters such as Scrollsage Nola, Loh, Ludwig von Tortollan, etc.
  - [x] Loh
  - [x] Ludwig von Tortollan
  - [x] Spiritsinger Umbra
  - [x] Scrollsage Nola
- [x] Assign scripted appearances to important Tortollan characters.

## ⚠️ Known issues:

- [x] 1. The **_Tortollan_** counties have some humans with Tortollan culture spawning in without the creature traits.

<details><summary>Click to expand</summary>

image

</details>

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
1. Provide feedback on this addition. Consider things such as the location of Tortollan characters and territory (does it seem lore appropriate, might it skew game balance, etc.) as well as the look of the static models, trait localization, and anything else that you think might be worth mentioning.
2. Report any bugs if you run into them.

### CODE REVIEW

File: common/traits/wc_race_traits.txt [hunk 1]
```diff
@@ -2877,10 +2877,12 @@ creature_tortollan = {
 	sea_travel_danger = medium_sea_danger_reduction
 	coastal_sea_travel_danger = medium_sea_danger_reduction
 
+	same_opinion = 10
```
Comment (helloruhe) [inline common/traits/wc_race_traits.txt]: why do you keep adding this even tho it doesnt work

Comment (Grekopithikos) [reply common/traits/wc_race_traits.txt]: but I removed it

File: localization/english/names/wc_character_names_l_english.yml [hunk 2]
```diff
@@ -17721,4 +17721,149 @@
...
+ Veruca:0 "Veruca"
+ Yarga:0 "Yarga"
+ Zandle:0 "Zandle"
+ Zippy:0 "Zippy"
+ Zooka:0 "Zooka"
+ Cal'yoon:0 "Cal'yoon"
+ Kooley:0 "Kooley"
+ Masan'ka:0 "Masan'ka"
+ Paku'ai:0 "Paku'ai"
+ Zatanji:0 "Zatanji"
```
Comment (edmund-h) [inline localization/english/names/wc_character_names_l_english.yml:17869]: maybe add the names of the characters from WoW too so other characters with these names can be generated?

**Review by edmund-h — APPROVED** [review]

File: localization/english/names/wc_character_names_l_english.yml (hunk 2 above)
Comment (Grekopithikos) [reply localization/english/names/wc_character_names_l_english.yml:17869]: I've added the names of every tortollan in WoW in this list.
Did I miss anyone?

**Review by Myphicbowser — APPROVED** [review]
No errors in the slightest, the models look great, and I just love seeing all the turtles... this is wonderful

**Pull request merged**